config/standalone-webpack.config.js

```diff
@@ -0,0 +1,288 @@
+// Staging config. Also the default config that prod and dev are based off of.
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const ManifestPlugin = require('webpack-manifest-plugin');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer')
+  .BundleAnalyzerPlugin;
+const TerserPlugin = require('terser-webpack-plugin');
+const webpack = require('webpack');
+const path = require('path');
+const ENVIRONMENTS = require('../src/site/constants/environments');
+const BUCKETS = require('../src/site/constants/buckets');
+
+const {
+  getAppManifests,
+  getWebpackEntryPoints,
+} = require('../src/site/stages/build/webpack/manifest-helpers');
+
+require('@babel/polyfill');
+
+const timestamp = new Date().getTime();
+
+const getAbsolutePath = relativePath =>
+  path.join(__dirname, '../', relativePath);
+
+const globalEntryFiles = {
+  polyfills: getAbsolutePath('src/platform/polyfills/preESModulesPolyfills.js'),
+  style: getAbsolutePath('src/platform/site-wide/sass/style.scss'),
+  styleConsolidated: getAbsolutePath(
+    'src/applications/proxy-rewrite/sass/style-consolidated.scss',
+  ),
+  vendor: [
+    getAbsolutePath('src/platform/polyfills'),
+    'react',
+    'react-dom',
+    'react-redux',
+    'redux',
+    'redux-thunk',
+    '@sentry/browser',
+  ],
+};
+
+/**
+ * Get a list of all the entry points.
+ *
+ * If the `entry` CLI argument is passed, only the specified
+ * application entries are built.
+ */
+function getEntryPoints(entry) {
+  const manifests = getAppManifests(path.join(__dirname, '../'));
+  let manifestsToBuild = manifests;
+  if (entry) {
+    const entryNames = entry.split(',').map(name => name.trim());
+    manifestsToBuild = manifests.filter(manifest =>
+      entryNames.includes(manifest.entryName),
+    );
+  }
+
+  return getWebpackEntryPoints(manifestsToBuild);
+}
+
+module.exports = env => {
+  const buildOptions = Object.assign(
+    {},
+    {
+      buildtype: 'localhost',
+      api: '',
+    },
+    env,
+  );
+  // Assign additional defaults which reference other properties
+  Object.assign(buildOptions, {
+    destination: path.resolve('../', 'build', buildOptions.buildtype),
+  });
+  const apps = getEntryPoints(buildOptions.entry);
```

Comment:
I'm calling `getEntryPoints` in here instead of passing the result in, like we're doing in the [original config file](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/webpack/index.js#L37).